themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +35,28 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group'] && strtolower($edsConfig->AuthorResultListInfo->ResultListPage ?? 'Long') == 'short'): ?>
+                    <?php $authors = $driver->getPrimaryAuthorsWithHighlighting(); ?>
+		    <?php $authorCount = count($authors) ?>
+		    <?php $authorNameLimit = $edsConfig->AuthorResultListInfo->AuthorNameLimit ?? 3; ?>
+                    <?php if (!empty($authors)): ?>
+                        <b><?=$this->transEsc($item['Label'])?>:</b>
```

Comment:
I notice that the else clause is inside a `<p>` tag but this case is not. Is that intentional, or should the `<p>` be added to match?

Reply:
thanks @demiankatz ! Ive included that in my latest follow up